db/re/item_db.conf

```diff
@@ -76293,6 +76435,24 @@ item_db: (
 	Script: <" itemheal rand(20,30),0; ">
 },
 {
+	Id: 11557
+	AegisName: "TE_White_Potion"
+	Name: "TE White Potion"
+	Type: 0
+	Buy: 200
+	Weight: 300
+	Script: <" itemheal rand(325,405),0; ">
```

Comment:
I can't use `Woe_White_Potion` in prontera town, ok
but I can use `TE_White_Potion` in prontera town, is this official ?